Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -65,6 +65,7 @@ type Props = NavigationSceneRendererProps & {
   renderRightComponent: NavigationSceneRenderer,
   renderTitleComponent: NavigationSceneRenderer,
   style?: any;
+  subviewStyle: any;
```

Comment:
`subviewStyle: any;` means it can be of any type, but it's compulsory. Adding a `?` as `style?: any` has will mark it as optional.